controller/chart/visualize.php

```diff
@@ -90,6 +90,17 @@
                     return ['id'=>$t->getId(), 'title'=>$t->getTitle()];
                 }, ThemeQuery::create()->allThemesForUser($chart)),
         );
+
+        try {
+            if ($chart->getMetadata('data.external-metadata')) {
```

Comment:
in the other PR you also checked that upload-method is "external-data". Guess it would be good to be consistent with the conditions.

Reply:
makes sense, updated